src/dash-table/derived/cell/wrappers.tsx

```diff
@@ -19,40 +19,76 @@ class Wrappers {
 
     }
 
+    partialGet = memoizeOne((
+        columns: VisibleColumns,
+        data: Data,
+        _offset: IViewportOffset
```

Comment:
`data` needs to come with `offset` for cache busting purposes but it's not used here. Prefixing with `_` disables parameter usage check with the current TS config